distribution/test/itests/test-itests-common/src/main/java/org/codice/ddf/itests/common/AbstractIntegrationTest.java

```diff
@@ -270,6 +270,15 @@ public String toString() {
 
     public static final DynamicPort RMI_REG_PORT = new DynamicPort(5);
 
+    public static final AbstractIntegrationTest.DynamicPort AMQP_PORT =
```

Comment:
We could use lower port numbers but I don't think this should be an issue. Have we run into cases where we think this would be a problem?